docs/embark-anyone.rst

```diff
@@ -0,0 +1,18 @@
+embark-anyone
+=============
+
+.. dfhack-tool::
+    :summary: Allows you to embark as any civilisation, including dead and non-dwarven ones.
+    :tags: embark gameplay armok
```

Comment:
TODO: think about the implications of combining `gameplay` and `armok`

(If I submit this review without finishing my thoughts here, please poke me about this)

Reply:
more words:

I've been treating "gameplay" and "armok" as mutually exclusive, with "gameplay" meaning "changes the gameplay in some consistent way that may not be achievable in vanilla but is not inherently "cheaty" or game-breaking" and armok meaning "no restrictions -- the world conforms to your will".

In other words, "gameplay" tools can have some "armoky" elements to them, but they are intended to be a balanced extension to the world.

This script straddles that line closer than most other tools.

Both `unretire-anyone` and `gui/embark-anywhere` have the `armok` tag and not the `gameplay` tag, so my suspicion is that we should go with `:tags: embark armok` here and remove the `gameplay` tag.

Reply:
Given that some civs are borderline unplayable, I'll agree that armok alone is appropriate.